gpib_server.py

```diff
@@ -112,7 +112,7 @@ def refreshDevices(self):
                     elif addr.startswith('TCPIP'):
                         instName = addr
                     elif addr.startswith('USB'):
-                        instName = addr + '::INSTR'
+                        instName = addr
```

Comment:
Is the problem that `addr` already includes `::INSTR`, so adding this again is not needed? Or is the problem that the instrument identifier doesn't need the suffix `::INSTR` at all?

If we do this, then the three branches can all be collapsed into check:
```python
KNOWN_DEVICE_TYPES = ('GPIB', 'TCPIP', 'USB')
...
if not addr.startswith(KNOWN_DEVICE_TYPES):
    continue
instr = rm.get_instrument(addr)
...
```